docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.3'
+services:
+  playground:
+    image: "srcd/gitbase-playground"
+    ports:
+     - "8080:8080"
+    environment:
+      - GITBASEPG_ENV=dev
+      - GITBASEPG_DB_CONNECTION=gitbase@tcp(gitbase:3306)/none?maxAllowedPacket=4194304
+  gitbase:
+    image: "srcd/gitbase"
+    environment:
+      - BBLFSH_ENDPOINT=bblfsh:9432
+    volumes:
+     - ${REPOS_FOLDER}:/opt/repos
+  bblfsh:
+    image: "bblfsh/bblfshd"
+    privileged: true
+    volumes:
+     - /var/lib/bblfshd:/var/lib/bblfshd
```

Comment:
It wouldn't work on macos. Please consider using named volume instead:
https://github.com/bblfsh/bblfshd/issues/136